experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -186,14 +187,26 @@ shared_params = SharedCanopyParameters{FT, typeof(earth_param_set)}(
 )
 
 canopy_model_args = (; parameters = shared_params, domain = canopy_domain)
+
+# Snow model
+ρ_snow = FT(300.0)
+α_snow = FT(0.8)
```

Comment:
This is just the default albedo value, so I don't think we need to pass it in.

Also, why not just use the default density?

Reply:
fair enough! I will use the defaults.